pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -435,6 +435,23 @@ def test_to_json_categorical_index(self):
 
         assert result == expected
 
+    @pytest.mark.parametrize("name", [None, "foo"])
+    def test_multiindex_single_level(self, name):
+        # GH29928
+        index = pd.Index([1, 2, 3, 4], name=name)
```

Comment:
can you use a fixed json for the expected rather than constructing it

Reply:
Yes